system-x/services/fhir/src/main/java/software/tnb/fhir/service/Fhir.java

```diff
@@ -0,0 +1,25 @@
+package software.tnb.fhir.service;
+
+import software.tnb.common.service.Service;
+
+import java.util.Map;
+
+public abstract class Fhir implements Service {
```

Comment:
also implement `WithDockerImage` and remove the `FHIR_IMAGE_KEY` contant and move `FHIR_IMAGE` into the method from `WithDockerImage`